src/frontend/src/pages/GanttPage/GanttPageFilter.tsx

```diff
@@ -0,0 +1,197 @@
+              icon={<LooksOneOutlinedIcon />}
+              checkedIcon={<LooksOneOutlinedIcon />}
+              onChange={car1Handler}
+              sx={{
+                color: 'white',
+                '&.Mui-checked': {
+                  color: '#ef4345'
+                },
+                '& .MuiSvgIcon-root': {
+                  fontSize: '2rem'
+                }
+              }}
+            />
+          </Grid>
+          <Grid item>
+            <Checkbox
+              defaultChecked
+              icon={<LooksTwoOutlinedIcon />}
+              checkedIcon={<LooksTwoOutlinedIcon />}
+              onChange={car2Handler}
+              sx={{
+                color: 'white',
+                '&.Mui-checked': {
+                  color: '#ef4345'
+                },
+                '& .MuiSvgIcon-root': {
+                  fontSize: '2rem'
+                }
+              }}
+            />
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid item xs={12} md={2}>
+        <FormControl sx={{ width: '100%' }}>
+          <FormLabel>Status</FormLabel>
+          <Select value={status} onChange={statusHandler}>
+            <MenuItem value="All Statuses">All Statuses</MenuItem>
+            {Object.values(WbsElementStatus).map((status) => {
+              return <MenuItem value={status}>{status}</MenuItem>;
+            })}
+          </Select>
+        </FormControl>
+      </Grid>
+      <Grid item xs={12} md={2}>
+        <FormControl sx={{ width: '100%' }}>
+          <FormLabel>Team</FormLabel>
+          <Select value={selectedTeam} onChange={teamHandler}>
+            <MenuItem value="All Teams">All Teams</MenuItem>
+            {teamList.map((team) => (
+              <MenuItem value={team}>{team}</MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+      </Grid>
+      <Grid item xs={12} md={2}>
+        <FormControl sx={{ width: '100%' }}>
+          <FormLabel>Start Date</FormLabel>
+          <DatePicker
+            inputFormat="yyyy-MM-dd"
+            onChange={startHandler}
+            className={'padding: 10'}
+            value={currentStart}
+            renderInput={(params) => <TextField autoComplete="off" {...params} />}
+          />
+        </FormControl>
+      </Grid>
+      <Grid item xs={12} md={2}>
+        <FormControl sx={{ width: '100%' }}>
+          <FormLabel>End Date</FormLabel>
+          <DatePicker
+            inputFormat="yyyy-MM-dd"
+            onChange={endHandler}
+            className={'padding: 10'}
+            value={currentEnd}
+            renderInput={(params) => <TextField autoComplete="off" {...params} />}
+          />
+        </FormControl>
+      </Grid>
+      <Grid
+        item
+        container
+        xs={12}
+        md={3}
+        spacing={1}
+        sx={{ justifyContent: 'end', alignItems: 'center', alignSelf: 'center', justifySelf: 'end' }}
+      >
+        <Grid item>
+          <Button
+            variant="contained"
```

Comment:
Can we have all these buttons be outlined or text instead of contained? I think contained is too much emphasis for these actions.